tests/scripts/github_tag_teams.py

```diff
@@ -244,6 +244,9 @@ def gen_cc_line(users):
     to_cc = [teams.get(t, []) for t in tags]
     to_cc = list(set(item for sublist in to_cc for item in sublist))
     to_cc = [user for user in to_cc if user != author]
+    existing_tags = list(set(re.findall(r"@[a-zA-Z0-9]+\w", body)))
```

Comment:
This misses `-` which can validly exist in a GitHub username (see https://github.com/paddy-hack and https://github.com/shinnn/github-username-regex), could you add it?

Also does this work if the `@Mousius` is right at the end of the line with no trailing whitespace?

Reply:
good catch, fixed both and added a test